content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -72,7 +72,7 @@ integrates with them, you must pass additional arguments to the
 {{< text bash >}}
```

Comment:
I know you didn't introduce this text, but it seems a little dubious.  The expectation here is that Kiali has already been installed (otherwise the jsonpath for clusterIP would not function properly).  Could you add a warning indicating that Istio and Kiali must already be installed?  Looking at the full file view, I see its sort of there already, but there isn't a strong warning - just an idea icon.

How does one install these components without first installing Istio, then reapplying an updated manifest?

Reply:
Good point @sdake !  I actually didn't use the clusterIP query because I was trying to apply this config at install time.  Using service names as host address was adequate.  I can clean that up to cover a compatible install time and post install update.

Reply:
@sdake There is a note at the begining of the document stating that: The following instructions assume you have installed Helm and use it to install Kiali.
I don't believe a further warning on that regard is needed.
The second question sound like a great suggestion for a related task, but I don't think it is within the scope of what this document is trying to do.